### DISCUSSION TITLE
Section 01: "PyTorch Workflow": Building a function to Visualize Our Data -plotting the data on the graph

### DISCUSSION THREAD

**Discussion by marvelousxmax** [mrdbourke/pytorch-deep-learning#919] [Q&A] [answered]
Hello! I am having trouble on video "Building a function to Visualize Our Data" (Video point 6:02). I have created the graph, but my labels are not appearing on the graph. I have provided three screenshots of the issue. Thank you.
![Screenshot 2024-05-06 1358523](https://github.com/mrdbourke/pytorch-deep-learning/assets/131393386/09b948e1-d9cc-46f9-bb39-f753b85dc405)

![Screenshot 2024-05-06 1356452](https://github.com/mrdbourke/pytorch-deep-learning/assets/131393386/e58c9a64-a4b1-4723-8e19-4c8215a59f41)

![Screenshot 2024-05-06 135451](https://github.com/mrdbourke/pytorch-deep-learning/assets/131393386/2b86bf8a-f14e-46f5-a4be-36969fa9f55b)

**Comment by IlanVinograd** [comment, marked as answer]
Hi there, your argument ` labels` is syntactically incorrect. You should use `label`. Let me know if this helps you! 😄
![Screenshot 2024-05-07 232107](https://github.com/mrdbourke/pytorch-deep-learning/assets/128734827/89e3fc40-3e63-4d36-95c7-d8df5e6dd7e9)

**Reply by marvelousxmax** [reply]
Thank you so much! That did the trick and the plots appeared!
